spec/ddtrace/contrib/rack/integration_test_spec.rb

```diff
@@ -123,7 +123,7 @@
             it do
               expect(span.name).to eq('rack.request')
               expect(span.span_type).to eq('web')
-              expect(span.service).to eq('rack')
+              expect(span.service).to eq(Datadog.configuration.service)
               expect(span.resource).to eq('GET 200')
               expect(span.get_tag('http.method')).to eq('GET')
               expect(span.get_tag('http.status_code')).to eq('200')
```

Comment:
Would it make sense to extract out these 6 checks? It seems like this boils down to "it's a rack GET 200" and a bunch of tests repeat them before actually getting to what's interesting and different in the test.

Reply:
Done!